drivers/mpu6050/MPU6050.cpp

```diff
@@ -138,7 +138,7 @@ struct fifo_packet {
 
 // Length of the FIFO used by the sensor to buffer unread
 // sensor data.
-#define MPU_MAX_LEN_FIFO_IN_BYTES 1024
+#define MPU_MAX_LEN_FIFO_IN_BYTES (4 * sizeof(fifo_packet))
```

Comment:
Sure that makes perfect sense. I'm just playing dumb reading the code and whatever isn't obvious, I ask. Can you add one small comment that says what you just explained please? Thanks.